docs/architecture/adr-011-configuration-translation-system.md

```diff
@@ -0,0 +1,46 @@
+# Architecture Decision Record: Configuration translation system
+
+## Context
+
+The translation system described in [adr-009-translation-system](./adr-009-translation-system.md) handles solely the translations added through Drupals traditional translation handling.
+
+But there was a wish for being able to translate configuration related strings as well: Titles, labels, descriptions etc. in the admin area.
+
+We needed to find a way to handle translation of that kind as well.
+
+## Decision
+
+We went for a solution where we activated the Configuration Translation Drupal core module and added the Configuration Translation PO contrib module.
+
+And we added a range of custom drush commands to handle the various configuration translation tasks.
+
+## Consequences
+
+By sticking to the handling of PO files in configuration handling that we are already using in our general translation handling, we can keep the current Github workflows with some alterations.
```

Comment:
Consider mentioning that using this approach handling translations of configuration on local sites is still as difficult as before. Translation of configuration texts cannot be found in the standard UI strings translation list.